action.yaml

```diff
@@ -0,0 +1,54 @@
+name: Bootstrap
+description: Bootstrap workflow setup for organization repositories
+author: Incognia
+inputs:
+  install-packages:
+    description: Packages to install
+    default: ''
+
+  setup-go:
+    description: Use inloco/actions-setup-go
+    default: ''
+
+  setup-java:
+    description: Use inloco/actions-setup-java
+    default: ''
+runs:
+  using: composite
+  steps:
+  - name: Export environment and mask secret variables
+    uses: inloco/actions-env-context@HEAD
+
+  - name: Install packages
+    shell: bash
+    if: inputs.install-packages
+    run: |
+      apt update
+      apt install -y ${{ inputs.install-packages }}
```

Comment:
Is apt's interface already stable? I remember that when using apt programmatically, it displayed an warning asking the user to use apt-get instead.